### DISCUSSION TITLE
About GitHub workflow error reporting

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#7008] [Q&A] [answered]
When I configured gh-pages, there was no problem with the build process, but at the end the following error occurred. Can you help me find out what happened?

```error
Push the commit or tag
  /usr/bin/git push origin gh-pages
  remote: Permission to W1ndys/ezqfnu.w1ndys.top.git denied to github-actions[bot].
  fatal: unable to access 'https://github.com/W1ndys/ezqfnu.w1ndys.top.git/': The requested URL returned error: 403
  Error: Action failed with "The process '/usr/bin/git' failed with exit code 128"
```

The following is additional information.My warehouse is open source. I don't know if you can visit it.

Workflow error page：https://github.com/W1ndys/ezqfnu.w1ndys.top/actions/runs/8531406406/job/23371649675

Workflow configuration file：https://github.com/W1ndys/ezqfnu.w1ndys.top/blob/main/.github/workflows/ezqfnu.yml

I am grateful for any help.

**Comment by w1ndys** [comment]
I tried deploying the key in the repository, but it didn't work

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @W1ndys,
the GITHUB_TOKEN is the default runner token you don't have to specify it explicitly:
https://github.com/marketplace/actions/github-pages-action

Since you're making changes to the same repository then you should have access to do it.

So the 403 error might be caused due to a lack of set permissions in the workflow file.

https://squidfunk.github.io/mkdocs-material/publishing-your-site/#__codelineno-0-7:8
